platform/ios/jazzy.yml

```diff
@@ -19,7 +19,7 @@ custom_categories:
     children:
       - Adding Markers to a Map
       - Runtime Styling
-      - Using Style Functions at Runtime
+      - Style Layers Using Expressions
```

Comment:
Avoid using “style” as a verb, because the reader is more likely to interpret it as a noun.

Much of this guide explains expressions by analogy to style function concepts, which makes it a poor introduction to expressions for readers who aren’t already familiar with style functions. “Predicates and Expressions” is meant to serve as a general-purpose introduction.

If any of the content in this guide is meant for a general audience, let’s move it to “Predicates and Expressions” and dedicate the rest of this guide to the task of migrating from the old syntax to the new syntax. The title could be “Upgrading Legacy Style Functions” or simply “Legacy Style Functions”.